README.rst

```diff
@@ -941,6 +941,15 @@ Choose from the list of available rules:
 
   The correct case must be used for standard PHP types in phpdoc.
 
+* **phpdoc_types_order**
+
+  Sorts PHPDoc types.
+
+  Configuration options:
+
+  - ``null_last`` (``bool``): whether ``null`` should be the last type; defaults to
```

Comment:
> what's the difference between not setting up null_position and setting it with default value ?

Not setting it would be the same as setting it to `always_first`, as it would be the default value of the option.

> imo:
> - sort_algorithm [opt]: 'alpha' (same, short value as in `ordered_imports` rule for consistency)
> - nuill_position [opt]: 'always_first, always_last'
> is enoguh for this PR. later, we could add other algorithms if needed

👍 (but still with `default` as possible value for `null_position`)